app/views/Dashboard/Overview/index.tsx

```diff
@@ -5,20 +5,17 @@ import {
     Tab,
     TabPanel,
     ContainerCard,
-    useModalState,
-    Button,
 } from '@the-deep/deep-ui';
 import { _cs } from '@togglecorp/fujs';
 
 import RegionalBreakdownCard from './RegionalBreakdownCard';
 import PercentageCardGroup from './PercentageCardGroup';
 import MapView from './MapView';
 import OverviewTable from './OverviewTable';
-import MapModal from './MapView/MapModal';
 import { FilterType } from '../Filters';
+import { TabTypes } from '..';
```

Comment:
same here

Reply:
This can be implemented because the file is only one step outside of it's file hierarchy.

Reply:
Yes, this is fine.